Sources/PrimerSDK/Classes/Core/PrimerHeadlessUniversalCheckout/Managers/Payment Method Managers/RawDataManager.swift

```diff
@@ -82,10 +82,16 @@ extension PrimerHeadlessUniversalCheckout {
         private var rawDataTokenizationBuilder: PrimerRawDataTokenizationBuilderProtocol
         public private(set) var paymentCheckoutData: PrimerCheckoutData?
         public private(set) var isDataValid: Bool = false
-        private var webViewController: SFSafariViewController?
+        var webViewController: SFSafariViewController?
         private var webViewCompletion: ((_ authorizationToken: String?, _ error: PrimerError?) -> Void)?
         var initializationData: PrimerInitializationData?
 
+        var tokenizationService: TokenizationServiceProtocol = TokenizationService()
+
+        var createResumePaymentService: CreateResumePaymentServiceProtocol
+
+        var apiClient: PrimerAPIClientXenditProtocol = PrimerAPIConfigurationModule.apiClient ?? PrimerAPIClient()
```

Comment:
This is very specific: `PrimerAPIClientXenditProtocol` - stands out in a generic manager.

Reply:
OK, I see now the `RawDataManager` supports `paymentCard`, `adyenBancontactCard` `xenditOvo` and `xenditRetailOutlets`, but only hits the api for `listRetailOutlets`. 🤯